src/main/java/com/epam/test_generator/controllers/GlobalExceptionController.java

```diff
@@ -19,6 +19,7 @@
 
     @ExceptionHandler(Exception.class)
     public ResponseEntity<Void> handleRunTimeException(Exception ex) {
+        ex.printStackTrace();
```

Comment:
R u sure that we need print stack trace ?

Reply:
done